cpp/CMakeLists.txt

```diff
@@ -338,16 +352,26 @@ message(STATUS "CUDF_TEST_LIST set to: ${CUDF_TEST_LIST}")
 # - add google benchmark --------------------------------------------------------------------------
 
 if(BUILD_BENCHMARKS)
+    find_package(benchmark)
 
-  include(ConfigureGoogleBenchmark)
+    if(NOT benchmark_FOUND)
+        include(ConfigureGoogleBenchmark)
+        set(benchmark_ROOT "${CMAKE_BINARY_DIR}/googlebenchmark")
+    endif(benchmark_FOUND)
+    add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
 
-  if(GBENCH_FOUND)
-    message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${GBENCH_ROOT}")
+    set(GBENCH_INCLUDE_DIR "${benchmark_ROOT}/install/include")
+    set(GBENCH_LIBRARY_DIR "${benchmark_ROOT}/install/lib")
+    link_directories(${GBENCH_LIBRARY_DIR}
     include_directories(${GBENCH_INCLUDE_DIR})
-    add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
-  else()
-    message(AUTHOR_WARNING "Google C++ Benchmarking Framework (Google Benchmark) not found: automated tests are disabled.")
-  endif(GBENCH_FOUND)
+    
+    if(benchmark_FOUND)
+        message(STATUS "Located Google C++ Benchmarking Framework (Google Benchmark) found in ${benchmark_ROOT}")
+    else():
```

Comment:
```suggestion
    else()
```

Reply:
Fixed in commit pushed at the same time you made this comment.